app/views/main.html

```diff
@@ -32,11 +32,14 @@
   <option value="3">Slide 4</option>
 </select>
 
-<simple-slider style="width:612px; height:612px" auto-play="false" change="{{actualslide}}">
+<button ng-click="slider.prev()">Prev Slide</button>
+<button ng-click="slider.next()">Next Slide</button>
```

Comment:
:+1: the example surely helps to show the intention of having the `slider-instance` attribute there :smile: